features/Visitor_can_view_specific_article.feature

```diff
@@ -6,18 +6,19 @@ Feature: Visitor can read a specific article
 
     Background:
         Given the following article exists:
-            | title   | description         | content                      | journalist |
-            | News    | This is a news      | I like to eat cheese!        | Jon        |
-            | Article | This is an  article | I do not like to eat cheese! | Greg       |
+            | title   | description        | content                      | journalist |
+            | News    | This is a news     | I like to eat cheese!        | Jon        |
+            | Article | This is an article | I do not like to eat cheese! | Greg       |
         And I visit the landing page
         And I click on 'News'
 
     Scenario: Visitor can see specific article
         And I should see 'News'
+        And I should see 'This is a news'
         And I should see 'I like to eat cheese!'
         And I should see 'Jon'
 
     Scenario: Visitor can navigate back to the landing page
```

Comment:
We agreed yesterday that we can remove this scenario

Reply:
Done